README.md

```diff
@@ -236,14 +236,32 @@ Awesome DeFi apps you can deploy on Akash
 - [Honeygain](honeygain)
 - [IPRoyal Pawns](iproyal-pawns)
 - [Iron Fish](iron-fish)
-- [MoneroOcean / auto-switching CPU miner with XMR payout](moneroocean)
+- [MoneroOcean CPU with XMR payout](moneroocean)
 - [PacketStream](packetstream)
 - [Peer2Profit](peer2profit)
 - [pkt.cash](pkt-miner)
 - [RainbowMiner](rainbowminer)
 - [Raptoreum](raptoreum-miner)
 - [Traffmonetizer](traffmonetizer)
-- [xmrig](xmrig)
+- [XMRig CPU](xmrig)
+
+### Mining - GPU
```

Comment:
We already have this section.

Reply:
In search in Cloudmos it's a problem, you can't distinguish. It's important for seo as well. Please do not block because of this.

Reply:
Actually I need to add to RainbowMiner - should be there as well.

Reply:
![image](https://github.com/akash-network/awesome-akash/assets/3791945/f48b767b-b96e-410b-a05b-4327cc402c3c)